core/templates/dev/head/library/Library.js

```diff
@@ -79,7 +79,8 @@ oppia.controller('Library', [
 
       var windowWidth = $(window).width() * 0.85;
       $scope.tileDisplayCount = Math.min(
-        Math.floor(windowWidth / tileDisplayWidth), MAX_NUM_TILES_PER_ROW);
+        Math.floor(windowWidth / (tileDisplayWidth + 20)),
```

Comment:
I think what happened was that there was an extra padding of 2px on both sides (total of 4px more in margin), so we actually need to go over so that we can guarantee it will render correctly. Because the md-card being inside the exploration card tag, it doesn't show up nicely in Inspect elements.